.gitignore

```diff
@@ -0,0 +1 @@
+.dockerconfigjson
```

Comment:
What is this change for?

Reply:
@Jamstah I answered that [here](https://github.com/IBM/iam-to-keycloak-migration/pull/5#discussion_r1991238414)

Reply:
I'll resolve this one, as I see you already asked the same above, so I'll keep that one for the current question/change request